Utils/Dataflow/pyDKB/storages/exceptions.py

```diff
@@ -0,0 +1,25 @@
+"""
+pyDKB.storages.exceptions
+"""
+
+
+class StorageException(Exception):
+    """ Base exception for all storage-related exceptions. """
+    pass
+
+
+class NotFound(StorageException):
+    """ Exception indicating that record with given ID not found. """
```

Comment:
Same as `Client.get()` above, "ID WAS not found". When consensus will be reached, this should be changed (or left as-is) accordingly.

Reply:
@Evildoor, consensus takes at least a discussion, but I still don\`t know your opinion on [this comment](https://github.com/PanDAWMS/dkb/pull/277#discussion_r302506396) ;)